app/forms/state_file/id_disability_form.rb

```diff
@@ -21,23 +21,42 @@ def should_check_spouse_disabled?
 
     def save
       if intake.show_mfj_disability_options?
-        case mfj_disability
-        when "primary"
-          @intake.update(primary_disabled: "yes", spouse_disabled: "no")
-        when "spouse"
-          @intake.update(primary_disabled: "no", spouse_disabled: "yes")
-        when "both"
-          @intake.update(primary_disabled: "yes", spouse_disabled: "yes")
-        when "none"
-          @intake.update(primary_disabled: "no", spouse_disabled: "no")
-        end
+        mfj_disability_to_disabled_attributes = self.class.mfj_disability_to_disabled_attributes_hash[mfj_disability&.to_sym]
+
+        @intake.update(
+          primary_disabled: mfj_disability_to_disabled_attributes && mfj_disability_to_disabled_attributes[:primary],
```

Comment:
yes! removing the methods and doing it straight here and being able to see primary/spouse instead of 0/1 makes it so much clearer and easier to read